policy/policy.go

```diff
@@ -103,6 +103,14 @@ func (s Set) Get(p Policy) (string, bool) {
 	return v, ok
 }
 
+func (s Set) ToStringMap() map[string]string {
```

Comment:
Does `Set` not already marshal to JSON without intervention? The underlying type is `map[string]string` after all.